packages/deploy/deploy/500_marketplace/504_deploy_exchange.ts

```diff
@@ -7,9 +7,9 @@ const func: DeployFunction = async function (hre: HardhatRuntimeEnvironment) {
   const {deployer, sandAdmin, upgradeAdmin, exchangeFeeRecipient} =
     await getNamedAccounts();
 
-  let TRUSTED_FORWARDER = await deployments.getOrNull('TRUSTED_FORWARDER');
+  let TRUSTED_FORWARDER = await deployments.getOrNull('TRUSTED_FORWARDER_V2');
   if (!TRUSTED_FORWARDER) {
-    TRUSTED_FORWARDER = await deploy('TRUSTED_FORWARDER', {
+    TRUSTED_FORWARDER = await deploy('TRUSTED_FORWARDER_V2', {
```

Comment:
remove the deployment from here and add a dependency to TRUSTED_FORWARDER_V2 on the deploy script